pkg/openscap/openscap.go

```diff
@@ -173,10 +179,19 @@ func (s *defaultOSCAPScanner) Scan(mountPath string, image *docker.Image) error
 		return fmt.Errorf("Unable to retreive the CVE file: %v\n", err)
 	}
 
+	if s.HTML {
```

Comment:
nit: we can probably simplify this so we don't have to have the full command twice with something like:

```go
args []string = {"xccdf", "eval", "--results-arf", s.ResultsFileName()}

if s.Html {
  args = append(args, "--report", s.HTMLResultsFileName())
}

args = append(args, cveFileName)

_, err = s.chrootOscap(args...)
return err
```